### ISSUE TITLE
Annotation styling

### ISSUE THREAD

**Issue by elizabeth-dz** [ManifoldScholar/manifold#2751]
## Feature Description
More control over the appearance/styling of annotations. For example, being able to toggle on and off the date, changing the color of the underline or the "author" flag.

## *Why* is this feature important? *Who* does it help?
More customization of the annotations will make it possible to integrate the annotations more fully into the branding of the instance, so this feature is really for publishers.

## User Stories
In particular, I'd like to have a way to toggle off the date. When we at LQ are creating an official set of "author" annotations for a book, we'd like them to serve as the equivalent of footnotes in a scholarly edition (but more dynamic and easier to access than hyperlinked end notes in each essay/chapter). Be removing the date they would merge more seamlessly with the official text, instead of seeming like an add-on (also then no one can see how long it took me to add the annotations to the book).

## Design Notes

??

## Development Notes
I imagine that the easiest way to implement this would be to allow for publishers to add style sheets that can affect parts of the reader beyond the text itself, but I can also see why you might not want to do that/why it might not be ideal.

**Comment by zdavis** [comment]
Might be worth considering adding a new type of annotation for this use case, similar to what we do with resource annotations, rather than solving it by hiding dates. With resource annotations, we already have a paradigm for allowing an editorial user to add special annotations. We could create a new type of annotation that displayed differently as Elizabeth describes. Just thinking out loud...

**Comment by tsmyre** [comment]
I agree. I think that could be useful in a host of settings: authors wanting to guide discussions around their own materials, teachers leading and provoking discussion in classroom settings. Also kinda nice to extend that model in new ways.

**Comment by elizabeth-dz** [comment]
That makes sense to me! I imagine that solution would also make it easier to create greater visual distinction between publisher annotations and user annotations in other ways as well (more than just the underline color/author flag).

**Comment by zdavis** [comment]
Discussed this during our team meeting. We're going to do a little work on defining this further and will follow up with more details.

**Comment by zdavis** [comment]
@bot accept

**Comment by zdavis** [comment]
Thank you for taking the time to open this feature request. The Manifold team reviewed this issue during our bi-weekly meeting and the consensus is that this feature makes sense and is in keeping with our overall vision for the platform. Moreover, we see this request as a viable candidate for development under our current available funding. We’re adding an “accepted” label to this request to indicate that it’s within scope and possibly within budget.

The next step is for us to estimate the work involved with this and add it to our feature backlog. Our acceptance of the issue is not a promise that it will be implemented. We will balance this request against the other accepted requests and do our best to implement it within our current available funding.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by tsmyre** [comment]
I wonder if it would also be possible to use this model to create a bookmark annotation so readers could keep track of where they were in texts.
